src/api/app/views/webui/package/show.html.haml

```diff
@@ -80,8 +93,7 @@
       .card-body#comments
         = render partial: 'webui/comment/show', locals: { commentable: @package,
           comment_counter_id: "#comment-counter-package-#{@package.id}" }
-
-- if User.session
+- if !flipper_responsive? && User.session
```

Comment:
This is needed in both cases, so for the beta and non-beta UIs. Otherwise, modals are not working. Clicking on `Branch Package` in the `Actions` menu doesn't work right now. Nothing happens.